docs/datamodel.md

```diff
@@ -0,0 +1,60 @@
+## Example
+
```

Comment:
little bit raw. Requires some sentence / introduction what this chapter is about